src/js/stores/__tests__/ChronosStore-test.js

```diff
@@ -91,6 +233,77 @@ describe('ChronosStore', function () {
 
   });
 
+  describe('#monitorJobDetail', function () {
+
+    it('should call #fetchJobDetail with jobID in arguments', function () {
+      // Begin monitoring job details
+      ChronosStore.monitorJobDetail('foo');
+      expect(ChronosActions.fetchJobDetail).toHaveBeenCalledWith('foo');
+    });
+
+    it('should continuously poll for job details', function () {
+      // Begin monitoring job details
+      ChronosStore.monitorJobDetail('foo');
+      // Let three intervals run
+      jasmine.clock().tick(3 * Config.getRefreshRate());
+      // Finish up outstanding timers
+      jest.runOnlyPendingTimers();
+      expect(ChronosActions.fetchJobDetail.calls.count()).toEqual(4);
+    });
+
+    it('should continuously poll for multiple job details', function () {
+      // Begin monitoring job details
+      ChronosStore.monitorJobDetail('foo');
+      ChronosStore.monitorJobDetail('bar');
+      // Let three intervals run
+      jasmine.clock().tick(3 * Config.getRefreshRate());
+      // Finish up outstanding timers
+      jest.runOnlyPendingTimers();
+      expect(ChronosActions.fetchJobDetail.calls.count()).toEqual(8);
+    });
+
+  });
+
+  describe('#stopJobDetailMonitor', function () {
+
+    it('should prevent subequent fetchJobDetail calls for specific jobID',
+      function () {
+      // Begin monitoring job details on specific ID
+      ChronosStore.monitorJobDetail('foo');
+      // Let three intervals run
+      jasmine.clock().tick(3 * Config.getRefreshRate());
+      // Finish up outstanding timers
+      jest.runOnlyPendingTimers();
+      // Stop monitoring specific job's details
+      ChronosStore.stopJobDetailMonitor('foo');
+      // Initiate another 10 intervals to ensure the job is no longer fetched
+      jasmine.clock().tick(10 * Config.getRefreshRate());
+      // Finish up outstanding timers
+      jest.runOnlyPendingTimers();
+      expect(ChronosActions.fetchJobDetail.calls.count()).toEqual(4);
+    });
+
+    it('should prevent subequent fetchJobDetail calls for all jobID',
+      function () {
+      // Begin monitoring job details on specific IDs
+      ChronosStore.monitorJobDetail('foo');
+      ChronosStore.monitorJobDetail('bar');
+      ChronosStore.monitorJobDetail('baz');
+      // Let four intervals run
+      jasmine.clock().tick(4 * Config.getRefreshRate());
+      // Finish up outstanding timers
+      jest.runOnlyPendingTimers();
+      // Stop monitoring specific job's details
+      ChronosStore.stopJobDetailMonitor();
+      // Initiate another 10 intervals to ensure the jobs are no longer fetched
+      jasmine.clock().tick(10 * Config.getRefreshRate());
```

Comment:
#nit Could we make this just 1 more fetch, just so we end with just one more per job, i.e. `15 + 3 = 18`, if this should ever fail.
Right now if it will still fetch the count would be `15 + 33 = 48`